configs/groupfree3d/groupfree3d_8x4_sunrgbd-3d-10class-L6-O256.py

```diff
@@ -0,0 +1,177 @@
+_base_ = [
+    '../_base_/datasets/sunrgbd-3d-10class.py',
+    '../_base_/models/groupfree3d.py', '../_base_/schedules/schedule_3x.py',
+    '../_base_/default_runtime.py'
+]
+# model settings
+model = dict(
+    bbox_head=dict(
+        num_classes=10,
+        size_cls_agnostic=True,
+        bbox_coder=dict(
+            type='GroupFree3DBBoxCoder',
+            num_sizes=10,
+            num_dir_bins=12,
+            with_rot=True,
+            size_cls_agnostic=True,
+            axis_aligned_lw=True,
```

Comment:
Does the original implementation use the axis-aligned setting? Do we need to provide both settings with their benchmarks? Add more comments in this PR or codes.